### PR TITLE
Create new Organisation with 'LOCAL_AUTHORITY' org. type

### PR DESCRIPTION
**Pull request by sc15zs**
Description:
PR to add 'LOCAL_AUTHORITY' organisation_type when creating new Organisation on submission.

### CODE REVIEW

File: data_store/controllers/load_functions.py [hunk 1]
```diff
@@ -69,6 +69,7 @@ def load_organisation_ref(transformed_data: dict[str, pd.DataFrame], mapping: Da
         org_to_merge.id = organisation_exists.id
         db.session.merge(org_to_merge)
     else:
+        organisation.organisation_type = "LOCAL_AUTHORITY"
```
[the remark this answers is not in the document]
Comment (gidsg) [reply data_store/controllers/load_functions.py]: Like this? https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/f418298e05cf5ed35593461b952fe5b6039bb71d/data_store/db/entities.py#L169

[the remark this answers is not in the document]
Comment (sc15zs) [reply data_store/controllers/load_functions.py]: Yes, you're right. Changed it to that.

**Comment by gidsg** [comment]
Can we write a test for this? Although guess it's the same as the default behaviour anyway?

**Review by gidsg — APPROVED** [review]
LGTM 👍

**Pull request merged**